includes/admin/amp-wp-admin-styles.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+add_action('admin_head', 'amp_options_styles');
+function amp_options_styles() {
```

Comment:
(1) Agree. Moved.
(2) What is the best way to restrict it to the analytics options page? The page does not use a template; should I use the URL parameter 'page='amp-analytics'?